projects/Bactteria_Dataset/configs/Bactteria_Det_unet_0.0001_CrossEntropyLoss.py

```diff
@@ -0,0 +1,185 @@
+img_scale = (512, 512)
```

Comment:
Configs can be simplified by inheritance. Such as:

1. use ```_base_ = [ 'mmseg::_base_/default_runtime.py']``` as https://github.com/okotaku/clshub/blob/db569f355362cce6a0ebf010a3fbb1c1beafc33b/configs/projects/rsna2022/efficientnet/efficientnet-b3_2xb8_rsna2022.py#L5

2. Inheritance-related cfg
```
_base_ = [ './Bactteria_Det_unet_0.0001_CrossEntropyLoss.py']

optim_wrapper = dict(optimizer=dict(lr=0.1))  # only change the lr
```

3. delete Intermediate variable
such `train_pipeline`, `optimizer`. users will be confused if they modify the `train_pipeline` but have no effect.